Index: trumbowyg.js
===================================================================
--- trumbowyg.js	(revision 4207)
+++ trumbowyg.js	(working copy)
@@ -930,22 +930,13 @@
                     label: 'URL',
                     required: true
                 },
-                title: {
-                    label: t.lang.title
-                },
                 text: {
                     label: t.lang.text,
                     value: t.getSelectedText()
-                },
-                target: {
-                    label: t.lang.target
                 }
             }, function(v){ // v is value
                 var link = $(['<a href="', v.url, '">', v.text, '</a>'].join(''));
-                if (v.title.length > 0)
-                    link.attr('title',v.title);
-                if (v.target.length > 0)
-                    link.attr('target',v.target);
+                link.attr('target', '_blank');
                 t.selection.deleteContents();
                 t.selection.insertNode(link.get(0));
                 t.restoreSelection();
